### PR TITLE
final datatables for MP, PLoci, MLH and their relations

### LINKED ISSUE #224: Add three more yellow datatables after all related data curation issues are solved

### PR DESCRIPTION
**Pull request by Nuanda**
Fixes #224

### CODE REVIEW

**Comment by Nuanda** [comment]
@nowakowski Piotr, I merged your latest data fixes to this PR.

File: spec/models/plant_accession_spec.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@
     it 'will query by permitted params' do
       pas = create_list(:plant_accession, 2)
       filtered = PlantAccession.filter(
-        query: { id: pas[0].id }
+        query: { 'id' => pas[0].id }
```
Comment (nowakowski) [inline spec/models/plant_accession_spec.rb]: Why the old-fashioned notation?

Comment (Nuanda) [inline spec/models/plant_accession_spec.rb]: I decided to turn all query parameters to string - that was discussed with Tomek as well. It makes things a little bit cleaner. With strings, however, you can't use the new notations (which is suitable for symbols).

**Comment by nowakowski** [comment]
:+1:

**Pull request merged**